about.html

```diff
@@ -0,0 +1,171 @@
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <!-- <link rel="stylesheet" href="style.css" /> -->
+    <link rel="stylesheet" href="about.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <header class="toolbar">
+      <menu class="menu_items">
+        <p class="close_menu_icon">X</p>
+        <ul>
+          <li><a href="./index.html" tabindex="1">Home</a></li>
+          <li><a href="#" tabindex="2">About</a></li>
+          <li><a href="./index.html#main_program" tabindex="3">Program</a></li>
+          <li><a href="./index.html#speakers" tabindex="4">Speakers</a></li>
+          <li><a href="#sponsors" tabindex="5"> Sponsors</a></li>
+        </ul>
+      </menu>
+      <nav class="menu_icon"><i class="fa-solid fa-bars" id="menu_icon"></i></nav>
+      <div class="nav_items">
+        <div class="toolbar_top">
+          <ul class="socials">
+            <li class="icons icons1">
+              <a href="https://twitter.com/EzzyWealth"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li class="icons icons2">
+              <a href="https://www.linkedin.com/in/ezekiel-udiomuno-b14539150/">
+                <i class="fa-brands fa-linkedin-in"></i
+              ></a>
+            </li>
+
+            <li class="icons icons4">
+              <a href="https://github.com/Ezzywealth"
+                ><i class="fa-brands fa-github"></i
+              ></a>
+            </li>
+          </ul>
+          <ul>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+          </ul>
+        </div>
+        <div class="toolbar_bottom">
+          <a href="./index.html"
+            ><img src="./images/javascript-39424.png" alt="logo" class="logo"
+          /></a>
+
+          <nav>
+            <a href="./index.html" tabindex="1">Home</a>
+            <a href="#" tabindex="2">About</a>
+            <a href="./index.html#main_program" tabindex="3">Program</a>
+            <a href="./index.html#speakers" tabindex="4">Speakers</a>
+            <a href="#sponsors" tabindex="5"> Sponsors</a>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="headline_section">
+        <article class="about_headline_item">
+          <h1 class="headline_header about_header">
+            #1 Javascript <br />
+            Conference On <br />
+            the Planet
+          </h1>
```

Comment:
- [ ] To match the template design. I think it would be better if you update some styling of the page title such as: `upercase` text transformation, increase  a little bit the `font size`, `line height`,... The important is to make sure its styling is so close as  much as possible the template design